Content.Shared/Hands/Components/SharedHandsComponent.cs

```diff
@@ -1,11 +1,13 @@
+using Content.Shared.Hands.EntitySystems;
 using Robust.Shared.Containers;
 using Robust.Shared.GameStates;
 using Robust.Shared.Serialization;
 
 namespace Content.Shared.Hands.Components;
 
-[NetworkedComponent]
-public abstract class SharedHandsComponent : Component
+[RegisterComponent, NetworkedComponent]
+[Access(typeof(SharedHandsSystem))]
+public sealed class SharedHandsComponent : Component
```

Comment:
If it's only in shared just dump the Shared prefix.

Reply:
I didn't want to do it originally because of the amount of files it would change but done in [`af3982d` (#15197)](https://github.com/space-wizards/space-station-14/pull/15197/commits/af3982de4a843c2fd70ccf583e1c4ca1b84ba17e)